jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/registration.js

```diff
@@ -18,9 +18,19 @@
  * todo: split and refine it
  */
 
+/**
+ * This javascript called twice and i didn't find how to prevent this occurrence.
```

Comment:
To find that out you can throw an error and see the stack trace to identify places where this is called.